pkg/agent/route/interfaces.go

```diff
@@ -44,4 +44,7 @@ type Interface interface {
 	// UnMigrateRoutesFromGw should move routes back from local gateway to original device linkName
 	// if linkName is nil, it should remove the routes.
 	UnMigrateRoutesFromGw(route *net.IPNet, linkName string) error
+
+	// Run periodically syncs iptables.
```

Comment:
The interface is for windows too, could we avoid using linux specific concept like other methods.

Reply:
Updated to - "Run periodically syncs firewall rules required for routing in host network."